highfidelity/hifi
# Speed up Entity List properties fetching by adding Entities.getMultipleEntities API

## QA tests

1. Run the following test case: https://highfidelity.testrail.net/index.php?/tests/view/120401

2. Run the Client Script tests
   1. Enable Developer menu (Settings -> Developer Menu)
   2. Go to your sandbox or serverless-domain
   3. Run `Developer -> Tests -> Run Client Script Tests` ( a dialog appears)
   4. All the Entity tests should pass, you can ignore all the other unit tests for this QA. (if you are on serverless it is also expected that `Entity serversExist` will fail)
![image](https://user-images.githubusercontent.com/607735/46239052-59dc0380-c393-11e8-9e34-28a92574f775.png)


(OSX only update,  apparently Run Client Script Tests doesn't seem to work on the OSX,   open `Developer -> Scripting Console` instead and paste `Script.include("https://raw.githubusercontent.com/highfidelity/hifi/eeda7400c99b1c61472e06d9642ae93b1878e361/scripts/developer/tests/unit_tests/testRunner.js");` )
